### ISSUE TITLE
Remove Object From Track When Disabled in Renders

### ISSUE THREAD

**Issue by PBnFlash** [RayMairlot/Animated-Render-Border#21]
I believe the current behavior tracks all objects in a collection, regardless of their render state.
 
I am working on a project where it is convenient to enable/disable objects from the render, animated render boarder unfortunately will still render the empty pixels where these objects are. 

It would be nice to cull hidden objects from collections. It would save over half the render time in my case.

**Comment by RayMairlot** [comment]
Hi. I'll have to think about this and how it fits in with other features. I can see how it would be useful though.

Is adding the objects you want to render to a separate collection (in addition to their original collection) not possible in your workflow?

**Comment by PBnFlash** [comment]
Yes in my case they are all in the root of one collection with no sub collections. 
*edit* 
In my case I could add them to another collection to indicate I want this behavior. 

Ideally it would be a radio button in the toolbar toggles this behavior for all items in the collection.

**Comment by PBnFlash** [comment]
In the mean time I'm going to do this and hope it won't break anything.
`                objs = (object for object in bpy.data.collections[border.collection].objects if object.type in trackableObjectTypes) #Type of objects that can be tracked
`
To
`                objs = (object for object in bpy.data.collections[border.collection].objects if object.type in trackableObjectTypes and object.hide_render == False) #Type of objects that can be tracked
`
